### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [kirshr/TeaCozy#2]
You used alt text on some of the pictures but not all of them.

All images should have a descriptive alt text. Reason being is that the visually impaired use screen readers. The screen readers read the alt text to provide them a description of the image.
